db/snapcfg/util.go

```diff
@@ -509,6 +523,16 @@ func webseedsParse(in []byte) (res []string) {
 }
 
 func LoadRemotePreverified(ctx context.Context) (err error) {
+	if s, ok := os.LookupEnv("ERIGON_REMOTE_PREVERIFIED"); ok {
```

Comment:
a bit bad naming - because unclear to which part of erigon it related

Reply:
Yeah I wasn't sure. DOWNLOADER didn't seem right either, but neither did a naked env key. What do you suggest? `SNAPCFG_REMOTE_PREVERIFIED`?

Reply:
I'll wait for a suggestion on naming before I merge